src/Firestore/V1beta1/FirestoreAdminClient.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/*
+ * Copyright 2017, Google Inc. All rights reserved.
```

Comment:
We decided to not ship the AdminClients since they have received very little testing.

Reply:
We just discussed this (David, Vadym and I) yesterday. This client will be removed before the PR is merged.